spec/acceptance/server_instance_spec.rb

```diff
@@ -3,7 +3,7 @@
 # run a test task
 require 'spec_helper_acceptance'
 
-describe 'postgresql instance test1', if: os[:family] == 'redhat' && os[:release].start_with?('8') do
+describe 'postgresql instance test1', if: os[:family] == 'redhat' && !os[:release].start_with?('7') do
   pp = <<-MANIFEST
   # set global defaults
   class { 'postgresql::globals':
```

Comment:
I'd change the version to 15 or 16. I don't have a RHEL 8 at hand to see if it ships 16 already, so I'd suggest 15.

Reply:
```
Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
Name                                                                     Stream                                                               Profiles                                                                        Summary                                                                                         
postgresql                                                               9.6                                                                  client, server [d]                                                              PostgreSQL server and client module                                                             
postgresql                                                               10 [d]                                                               client, server [d]                                                              PostgreSQL server and client module                                                             
postgresql                                                               12                                                                   client, server [d]                                                              PostgreSQL server and client module                                                             
postgresql                                                               13                                                                   client, server [d]                                                              PostgreSQL server and client module                                                             
postgresql                                                               15                                                                   client, server [d]                                                              PostgreSQL server and client module                                                             
postgresql                                                               16                                                                   client, server [d]                                                              PostgreSQL server and client module                                             
```

Reply:
@bastelfreak could you change line 15 from `'13'` to `'16'`? I'd then expect it to pass.

Reply:
done, let's see.